cla_public/templates/base.html

```diff
@@ -54,38 +54,31 @@
       {% endfor %}
     {% endif %}
   {% endif %}
-  <div class="laa-flee-button-container
+  <main class="govuk-main-wrapper " id="main-content" role="main" lang="{{ request.cookies.get('locale', 'en')[:2] }}">
     {% if is_quick_exit_enabled(session) %}
```

Comment:
This block of code, from line 58 - 68 should be moved outside the `main-content` element, otherwise the button gets pushed down the screen.